heareval/train_baseline.py

```diff
@@ -0,0 +1,164 @@
+"""
+Train the baseline model on a single multiclass task.
+
+TODO: Some of the boilerplate code is shared with task_embeddings.py
+and can be cleaned up.
+"""
+
+import os.path
+from importlib import import_module
+
+import numpy as np
+import pytorch_lightning as pl
+import soundfile as sf
+import torch
+from torch.utils.data import DataLoader, random_split
+
+import heareval.baseline
+
+from csvdataset import CSVDataset
+from heareval.baseline import RandomProjectionMelEmbedding
+
+
+TASK = "coughvid-v2.0.0"
+
+# For this task
+# TODO: Don't hardcode
+FRAME_RATE = 8
+SAMPLE_LENGTH_SECONDS = 8.0
+N_LABELS = 3
+
+# TODO: Support for multiple GPUs?
+device = "gpu" if torch.cuda.is_available() else "cpu"
+
+# +
+VALIDATION = 0.2
+
+AUDIO_BATCH_SIZE = 64
+FRAME_BATCH_SIZE = 64
+EMBEDDING_SIZE = 4096
+
```

Comment:
We should be able to run this both in fine-tuning mode, and non-fined-tuned prediction mode.